packages/theme/src/types.ts

```diff
@@ -47,6 +47,7 @@ type PaletteAdditions = {
     background: string;
     indicator: string;
     color: string;
+    divider: string;
```

Comment:
This is a breaking change, and I'm not quite sure what to do with it tbh. The theme package has [stability index 2](https://backstage.io/docs/overview/stability-index#section-typescript), meaning we can't ship a breaking change without a deprecation period. What we'd need to do is to make it optional an then print a deprecation warning if it's not set, and then make it required in a future change.

What do you think about skipping this addition and instead using darkening the `theme.navigation.color` to match instead?